softwareComponents/configuration/readme.md

```diff
@@ -65,4 +66,4 @@ See examples in the `../data` directory.
 
 The configuration is represented by the `Configuration` class. See its definiton
 for more usage. The serialization/deserialization functions ara available in
-`IO.h`.
\ No newline at end of file
+`IO.h`.
```

Comment:
Toto je pravda pro legacy konfiguraci. Ale opravu dokumentace bych asi nechal na další PR. Co říkáš?

Reply:
Byla snaha přejmenovat názvy v dokumentaci, tohle je asi dobrý změnit - ale může to být v jiným PR

Reply:
Tak ne, je to v sekci o staré konfiguraci

Reply:
Ah, má chyba.